### ISSUE TITLE
Add missing focus indicators

### ISSUE THREAD

**Issue by HenrikHartiala** [HSLdevcom/jore4#1443]
Part of accessibility audit findings.

Some of these might be fixed by adding the focus indicator to the "base" button. 

- Focus problems. Fix: global border on focus in CSS?
     - Missing visual focus indicator
         - [ ] Header buttons (e.g. Profile button) ([13](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g24d8c9308d3_0_9))
         - [ ] (Routes) Control buttons ([22](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g24d8c9308d3_0_55))
         - [ ] (Routes) "Tarkenna" dropdown menus ([22](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g24d8c9308d3_0_55))
         - [ ] (Map) Buttons around map are missing focus indicators ([49](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g1e3be12a1f7_0_110))
         - [ ] (Forms) Toggle buttons ([60](https://docs.google.com/presentation/d/1ePFbPmTUUK8m5-11lZZMlUyMNefIVyOrLa_SrK17pCc/edit#slide=id.g250d9bedbc5_0_26))
